File: 0001 Code Chef/0015 Election Hopes/implementation/0005 kotlin/ElectionHopes.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    if(stringInput != null){
        var stringArray:Array<String> = stringInput.split(" ").toTypedArray();
        var x:Int? = stringArray[0].toIntOrNull();
        var y:Int? = stringArray[1].toIntOrNull();
        if(x != null && y != null){
            if(x >= 2*y){
                println("YES");
            }else{
                println("NO");
            }
        } 
    }
}

File: 0001 Code Chef/0015 Election Hopes/implementation/0003 c#/ElectionHopes.cs
using System;

class ElectionHopes{
    static void Main(string[] args){
        string stringInput = Console.ReadLine();
        string[] stringArray = stringInput.Split(" ");
        int x = Convert.ToInt32(stringArray[0]);
        int y = Convert.ToInt32(stringArray[1]);

        if(x >= 2*y){
            Console.WriteLine("YES");
        }else{
            Console.WriteLine("NO");
        }
    }
}